src/Components/BasePage.js

```diff
@@ -2,31 +2,34 @@ import React from "react";
 import Switch from '@mui/material/Switch';
 import Signup from "./Signup";
 import Login from "./Login";
+import Card from "./Card"
+import "./BasePage.scss";
 import { useState } from "react";
```

Comment:
Nitpick: It's a good idea to organize imports. I generally follow the following import order:
1. react or dependencies at top.
2. React related libraries
3. Other libraries
4. Your own code for other components
5. your own code for general JD
6. The stylesheet at the bottom

Reply:
*JS